tests/unit/plugins/janitor/test_authority.py

```diff
@@ -139,12 +141,28 @@ def _create_fake(klass, *args, **kwargs):
     assert expected_msg == (await rrset_channel.get())
     # run also calls self.cleanup at the end
     assert (await rrset_channel.get()) is None
+    gce_authority.metrics.timer.assert_called_once_with(
+        'janitor-authority-runtime')
+    gce_authority.metrics.timer(None).start.assert_called_once_with()
```

Comment:
This works because we've mocked timer to return the same mock object every time?  I'm having a little trouble finding the metrics definition to be sure.